server/vulnerabilities/oval/oval_platform.go

```diff
@@ -74,6 +83,12 @@ func (op Platform) ToGovalDictionaryFilename() string {
 	return fmt.Sprintf("%s_%s.sqlite3", GovalDictionaryFilePrefix, op)
 }
 
+// ToDownloadedFilename returns the filename of the sqlite3 files downloaded using
+// the goval-dictionary fetch method in the vulnerabilities generate-cve.yml workflow
+func (op Platform) ToDownloadedFilename() string {
```

Comment:
Since this is goval-dictionary specific, let's name it as such. Maybe `GovalDatabaseFilename`?